.github/workflows/tests.yml

```diff
@@ -20,7 +20,7 @@ jobs:
     runs-on: ubuntu-latest
     strategy:
       matrix:
-        python-version: [ '3.9', '3.10', '3.11', '3.12' ]
```

Comment:
Dropping 3.9 from the tests matrix due to this issue - https://github.com/pydantic/pydantic-settings/issues/298
We'd need to revisit this at a later time.